src/massive/munit/Assert.hx

```diff
@@ -226,4 +226,25 @@ class Assert
 	{
 		throw new AssertionException(msg, info);
 	}
+	
+	/**
+	 * Assert that an expectation was thrown. Can expect strings and non-strings.
+	 *
+	 * @param   code				a function which should throw an exception 
+	 * @return					  the exception that was thrown	  
+	 * @throws  AssertionException  if no expectation is thrown
+	 */
+	public static function throws(code:Dynamic, ?info:PosInfos):Dynamic
+	{
+		try
+		{
+			code();
+			fail("Expected exception wasn't thrown!", info);
+			return null; // needed to compile
+		}
+		catch (e:Dynamic)
```

Comment:
This still suffers the same issue, as you're catching everything that's thrown. I think we should be explicitly asserting that an expected error was thrown.

```haxe
function throws(expected:Dynamic, func:Dynamic, ?info:PosInfo)
{
    var actual = null;
    try func()
    catch(e:Dynamic) actual = e;
    if (Std.is(expected, String)) Assert.areEqual(expected, actual, info);
    else Assert.isType(actual, expected, info);
    return actual
}
```

```haxe
Assert.throws('some string', function() { throw 'some string'; });
Assert.throws(SomeError, function() { throw new SomeError(); });
```